drivers/watchdog/wdt_iwdg_stm32.c

```diff
@@ -24,14 +25,18 @@
 #define IWDG_RELOAD_MIN		(0x0000U)
 #define IWDG_RELOAD_MAX		(0x0FFFU)
 
+#ifndef STM32_LSI_CLOCK
+#define STM32_LSI_CLOCK LSI_VALUE
+#endif /* STM32_LSI_CLOCK */
+
```

Comment:
This default value correspond to #else in stm32_clock_control.h

Reply:
Right, let's centralize symbol definitions in one single file